### DISCUSSION TITLE
Dropdown inconsistency between the Select and the Combobox

### DISCUSSION THREAD

**Discussion by colinlienard** [chakra-ui/zag#1825] [Q&A]
I was wondering why the Select and the Combobox have different default positioning settings:

```json5
// Select
positioning: {
  placement: "bottom-start",
  gutter: 8,
  ...ctx.positioning,
},
```

```json5
// Combobox
positioning: {
  placement: "bottom",
  flip: false,
  sameWidth: true,
  ...ctx.positioning,
},
```

Especially with `flip` that is set to false with the combobox, which I think should be true by default because it is just a better behaviour, and it would be nice if similar components have similar behaviour

**Comment by segunadebayo** [comment]
Feel free to open a PR to remove the flip property and align the defaults.
